src/main/haskell/kore/test/Test/Kore/Builtin/Krypto.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE MagicHash #-}
```

Comment:
I don't think this module needs this language extension.

Reply:
Weird, I don't think I added that, must've been from an old commit